docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: '2'
+services:
+  web:
+    build: .
+    command: npm start
+    ports:
+      - "4567:4567"
+    volumes:
+      - .:/braintree_express_example
+    environment:
```

Comment:
I noticed this is the only docker integration that includes this environment variable. Was curious if this is something different that only express needs.